test/dev-hook.jsx

```diff
@@ -1,6 +1,6 @@
 import "react-hot-loader";
 import { hot } from "react-hot-loader/root";
-import DevLayout from "./examples/0-showcase.jsx";
+import DevLayout from "./examples/17-resizable-handles.jsx";
```

Comment:
Probably unintended?

Reply:
Apologies, was just used for local testing. Will revert.